OneSignalSDK/onesignal/src/main/java/com/onesignal/OneSignal.java

```diff
@@ -181,6 +181,10 @@ public boolean isAppClose() {
        */
       void tagsAvailable(JSONObject tags);
    }
+   
+   public interface GetNotificationsCountHandler {
```

Comment:
Since we released the major release, can you follow the new name style? rename GetNotificationsCountHandler to OSNotificationsCountHandler

Reply:
Done